### ISSUE TITLE
tern 0.22.3 crashes when typing export

### ISSUE THREAD

**Issue by abepark01** [ternjs/tern#996]
Node: 8.12.0
npm: 6.4.1
Tern Version:  0.22.3
In my file, typing the word `export` then space in a JS file seems to crash tern

```
Exception in thread Request Completion:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/admin/.vim/bundle/deoplete-ternjs/rplugin/python3/deoplete/sources/ternjs.py", line 318, in completation
    data = self.run_command(command, pos)
  File "/Users/admin/.vim/bundle/deoplete-ternjs/rplugin/python3/deoplete/sources/ternjs.py", line 259, in run_command
    data = self.make_request(doc, silent)
  File "/Users/admin/.vim/bundle/deoplete-ternjs/rplugin/python3/deoplete/sources/ternjs.py", line 227, in make_request
    req = opener.open(self._url, payload)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 525, in open
    response = self._open(req, data)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 543, in _open
    '_open', req)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 1345, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 1320, in do_open
    r = h.getresponse()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response
```

Sample Code
```es6
import React from 'react';
import { titleize, humanize } from 'underscore.string';
import moment from 'moment';
import accounting from 'accounting';

export
// ^^^ crash

export const headerFormatter = (column, colIndex, { sortElement }) => (
  <div className="text-grey-darker text-sm">
    <strong>{column.text}</strong>
    {sortElement}
  </div>
);

export const dateFormatter = cell => (
  <span>
    {cell ? moment(cell).format('MM/DD/YYYY') : ''}
  </span>
);

export const priceFormatter = cell => (
  <span>
    ${cell}
  </span>
);

// ...

export const customerFormatter = (cell, row) => {
  let entityId = row._id;
  let customer;
  let field = row;
  if (cell && typeof cell === 'object') {
    field = cell;
    entityId = cell._id;
  }

  switch (field._type) {
    // code
    default:
      customer = `${field.first_name || ''} ${field.last_name || ''}`;
  }

  return (
    <div>
      <a
        className=""
        href={`/users/${entityId}`}
      >
        {titleize(humanize(customer))}
      </a>
    </div>
  );
};
```

**Comment by othree** [comment]
Hi, sorry I can't reproduce this issue.
If possible, you can try the latest revision (git clone this repo)
Or open a tern server manually to see what happened on the server side.

**Comment by othree** [comment]
or maybe you have a `.tern-project` file

**Comment by abepark01** [comment]
Thank you! @othree will have some more info ready shortly.

**Comment by abepark01** [comment]
**.tern-config**
```
{
  "libs": [
    "browser",
    "underscore",
    "jquery",
    "ecmascript"
  ],
  "plugins": {
    "node": {},
    "modules": {},
    "es_modules": {}
  },
  "ecmaVersion": 7
}
```

**Comment by othree** [comment]
I tried 

```
nvim@0.3.1
python@3.7.0
node@8.12.0
latest TernJS
```

Not use deoplete, but use tern_for_vim instead.

**Comment by othree** [comment]
Is your environment work on Tern@0.21 ?

**Comment by abepark01** [comment]
I am using Tern@0.22.3.

Somehow the error happens when I type the keyword `export`

In the code example above, typing `accounting.` did show the list of methods available.

**Comment by othree** [comment]
I can reproduce now.

```
export _
```
 `_` is cursor position.
Trigger complete suggestion at that position will crash Tern server.
Will fix at next release (0.23.0).

**Comment by abepark01** [comment]
Verified fix in tern@0.23
Thank you @othree
